mne/viz/backends/renderer.py

```diff
@@ -112,6 +114,11 @@ def set_3d_backend(backend_name, verbose=None):
         _reload_backend(backend_name)
         MNE_3D_BACKEND = backend_name
 
+    # Qt5 macOS 11 compatibility
+    if (backend_name == 'pyvista' and sys.platform == 'darwin' and
```

Comment:
I actually don't think the backend name matters, mayavi will have the same problem

Reply:
I've dropped the `pyvista` check in 77f36c9